clio-server/src/test/scala/org/broadinstitute/clio/server/service/MockIndexService.scala

```diff
@@ -0,0 +1,60 @@
+package org.broadinstitute.clio.server.service
+
+import akka.NotUsed
+import akka.stream.scaladsl.Source
+import org.broadinstitute.clio.server.{ClioApp, MockClioApp}
+import org.broadinstitute.clio.server.dataaccess.elasticsearch.{
+  ClioDocument,
+  ElasticsearchIndex
+}
+import org.broadinstitute.clio.transfer.model.TransferIndex
+import org.broadinstitute.clio.util.model.UpsertId
+
+import scala.collection.mutable.ArrayBuffer
+import scala.concurrent.{ExecutionContext, Future}
+import scala.reflect.ClassTag
+
+abstract class MockIndexService[
+  TI <: TransferIndex,
+  D <: ClioDocument: ElasticsearchIndex
+](
+  app: ClioApp = MockClioApp(),
+  override val transferIndex: TI
+)(
+  implicit
+  executionContext: ExecutionContext,
+  documentTag: ClassTag[D]
+) extends IndexService(
+      PersistenceService(app),
+      SearchService(app),
+      transferIndex
+    ) {
+  val queryCalls: ArrayBuffer[transferIndex.QueryInputType]
```

Comment:
Hrm, does assigning the `ArrayBuffer.empty[transferIndex.QueryInputType]` here (etc) not work?